csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/runtest-gpu.bat

```diff
@@ -3,19 +3,9 @@ REM Licensed under the MIT License.
 echo on
 
 set LocalNuGetRepo=%1
+SET CurrentOnnxRuntimeVersion=%2
 setlocal enableextensions disabledelayedexpansion
 
-REM WorkingDirectory is Build.SourcesDirectory\csharp
-set /p MajorVersionNumber=<..\VERSION_NUMBER
-set VersionSuffix=
-IF NOT DEFINED IsReleaseBuild (
-    FOR /F "tokens=* USEBACKQ" %%F IN (`git rev-parse --short HEAD`) DO (
-        set VersionSuffix=-dev-%%F
-    )
-)
-
-set CurrentOnnxRuntimeVersion=%MajorVersionNumber%%VersionSuffix%
-
 @echo %CurrentOnnxRuntimeVersion%
 
 pushd test\Microsoft.ML.OnnxRuntime.EndToEndTests.Capi
```

Comment:
The C# tests for GPU (in the CI build) seem to be getting skipped, even though TESTONGPU flag is set in **gpu-esrp-pipeline.yml**. May not be related to this change, but I believe they should be running.

```
[xUnit.net 00:00:02.19]     Microsoft.ML.OnnxRuntime.Tests.InferenceTest.TestGpu [SKIP]
```

Reply:
Thanks for noting this. Let me look into that issue as a separate one, This seems to be happening for a while before this change.